release/start-acs/patch-central.json

```diff
@@ -8,9 +8,15 @@
                         "env": [
                             {
                                 "name": "MUTEX_WATCHDOG_TIMEOUT_SECS",
-                                "value": "0",
+                                "value": "0"
+			    },
+			    {
                                 "name": "ROX_EXTERNAL_IPS",
                                 "value": "true"
+			    },
+			    {
+                                "name": "ROX_NETWORK_GRAPH_EXTERNAL_IPS",
```

Comment:
Not strictly needed, but does enable checking the external IPs in the network graph.